src/ReactiveUI.XamForms/ShellViewModel.cs

```diff
@@ -0,0 +1,89 @@
+﻿// Copyright (c) 2021 .NET Foundation and Contributors. All rights reserved.
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for full license information.
+
+using Splat;
+using Xamarin.Forms;
+
+namespace ReactiveUI.XamForms
+{
+    /// <summary>
+    /// ShellViewModel.
+    /// </summary>
+    /// <typeparam name="TViewModel">The type of the view model.</typeparam>
+    /// <seealso cref="ShellContent" />
+    public class ShellViewModel<TViewModel> : ShellContent, IActivatableView
```

Comment:
Is this a `View` or a `ViewModel`?

Reply:
Its a UI Element so View

Reply:
Renamed to ReactiveShellContent to make it clearer